src/main/java/com/treasure_data/td_import/Constants.java

```diff
@@ -95,6 +95,7 @@
             "    --column-types TYPE,TYPE,...     column types [string, int, long, double]\n" +
             "    --column-type NAME:TYPE          column type [string, int, long, double]. A pair of column name and type can be specified like 'age:int'\n" +
             "    -S, --all-string                 disable automatic type conversion\n" +
+            "    --empty-as-null                  the empty string values are interpreted as null values if columns arenumerical types." +
```

Comment:
```
Indicates that Amazon Redshift should load empty CHAR and VARCHAR fields as NULL. Empty fields for other data types, such as INT, are always loaded with NULL.
```
My understanding is 'emptyasnull' is for CHAR and VARCHAR (String) and it does not affect other data types such as INT, FLOAT, DOUBLE. That's exactly what I pointed that the name would not be good.